packages/fast-breakpoint-tracker-react/src/breakpoints.ts

```diff
@@ -0,0 +1,23 @@
+export type Breakpoint = number;
+
```

Comment:
I don't really think having a "Breakpoint" type is helpful. I think having an abstraction for a simple number just makes it harder to understand. Thoughts?

Reply:
Good with this. We can go with Breakpoints = number[];